DirectProgramming/DPC++FPGA/ReferenceDesigns/mvdr_beamforming/mvdr_beamforming.vcxproj

```diff
@@ -0,0 +1,185 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="Build" ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ItemGroup Label="ProjectConfigurations">
+    <ProjectConfiguration Include="Debug|x64">
+      <Configuration>Debug</Configuration>
+      <Platform>x64</Platform>
+    </ProjectConfiguration>
+    <ProjectConfiguration Include="Release|x64">
+      <Configuration>Release</Configuration>
+      <Platform>x64</Platform>
+    </ProjectConfiguration>
+  </ItemGroup>
+  <ItemGroup>
+    <ClCompile Include="src\mvdr_beamforming.cpp" />
+  </ItemGroup>
+  <ItemGroup>
+    <ClInclude Include="src\BackwardSubstitution.hpp" />
+    <ClInclude Include="src\Beamformer.hpp" />
+    <ClInclude Include="src\CalcWeights.hpp" />
+    <ClInclude Include="src\Constants.hpp" />
+    <ClInclude Include="src\FakeIOPipes.hpp" />
+    <ClInclude Include="src\Substitution.hpp" />
+    <ClInclude Include="src\LoopBackKernel.hpp" />
+    <ClInclude Include="src\mvdr_complex.hpp" />
+    <ClInclude Include="src\MVDR.hpp" />
+    <ClInclude Include="src\NullPipe.hpp" />
+    <ClInclude Include="src\pipe_array_internal.hpp" />
+    <ClInclude Include="src\pipe_array.hpp" />
+    <ClInclude Include="src\PipeAggregator.hpp" />
+    <ClInclude Include="src\SteeringVectorGenerator.hpp" />
+    <ClInclude Include="src\StreamingQRD.hpp" />
+    <ClInclude Include="src\Transpose.hpp" />
+    <ClInclude Include="src\UDP.hpp" />
```

Comment:
This line can also be deleted (file not added).

Reply:
fixed